example/pubspec.yaml

```diff
@@ -9,8 +9,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  google_atlas:
-    path: ../packages/google_atlas
+  google_atlas: ^0.0.6
```

Comment:
I think we can revert this change so that the example always uses the local copy